src/eduid_webapp/actions/app.py

```diff
@@ -56,14 +55,18 @@ def __init__(self, app):
                 self[plugin_name] = getattr(module, 'Plugin')
 
 
-def _get_tous(self, version=None):
+def _get_tous(app, version=None):
```

Comment:
It's actually a method - what I should have done is probably extend Flask to add the method. But outside of a class it looks better with app than with self, yes :)